test/integration/agent/ip_assigner_linux_test.go

```diff
@@ -29,7 +31,29 @@ import (
 const dummyDeviceName = "antrea-dummy0"
 
 func TestIPAssigner(t *testing.T) {
-	ipAssigner, err := ipassigner.NewIPAssigner(net.ParseIP("127.0.0.1"), dummyDeviceName)
+	linkList, err := net.Interfaces()
+	require.NoError(t, err)
+	getNodeIP := func() net.IP {
+		for _, link := range linkList {
+			if link.HardwareAddr == nil {
+				continue
+			}
+			addrList, err := link.Addrs()
+			if err != nil {
+				continue
+			}
+			for _, addr := range addrList {
+				if ipNet, ok := addr.(*net.IPNet); ok {
+					return ipNet.IP
+				}
+			}
+		}
+		return nil
+	}
+	nodeIP := getNodeIP()
```

Comment:
There is a global variable nodeIP (defined in route_test.go) can be used directly.

Reply:
Done